typescript-api/src/dancebox/interfaces/lookup.ts

```diff
@@ -380,53 +380,7 @@ export default {
             },
         },
     },
-    /** Lookup48: pallet_identity::pallet::Event<T> */
-    PalletIdentityEvent: {
```

Comment:
This is removing the pallet identity api, looks wrong

Reply:
yes, I should have opened this PR before pallet-identity was merged, but it is what it is

Reply:
I see